packages/vuetify/src/components/VDatePicker/VDatePicker.ts

```diff
@@ -134,25 +135,30 @@ export default mixins(
           return this.pickerDate
         }
 
-        const date = (this.multiple || this.range ? (this.value as string[])[(this.value as string[]).length - 1] : this.value) ||
+        const multipleValue = this.value ? wrapInArray(this.value) : []
```

Comment:
If we have access to `this` could we use `this.multipleValue`?

Reply:
computed accessed in data() returns undefined, `this.$options.computed.multipleValue.call(this)` works but it doesn't make sense